docs/e2e/e2e-checks.md

```diff
@@ -54,6 +56,20 @@ Then, run the tests with your app name and base url:
 make e2e-test-native APP_NAME=app BASE_URL=http://localhost:3000
 ```
 
+#### Running tests natively with shards
```

Comment:
use imperative voice consistently, and I suggest using terms people might search for rather than the underlying jargon that playwright uses
```suggestion
#### Run tests in parallel
```

Reply:
outdated diff but I updated this in a separate commit